pkg/cli/webhook.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"sigs.k8s.io/kubebuilder/pkg/plugin"
+)
+
+func (c cli) newCreateWebhookCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "webhook",
+		Short: "Scaffold a webhook for an API resource",
+		Long: `Scaffold a webhook for an API resource.
+`,
+		RunE: errCmdFunc(
+			fmt.Errorf("webhook subcommand requires an existing project"),
+		),
+	}
+
+	projectVersion, found := getProjectVersion()
+	if !found {
+		msg := `For project-specific information, run this command in the root directory of a
+project.
+`
+		cmd.Long = fmt.Sprintf("%s\n%s", cmd.Long, msg)
+		return cmd
```

Comment:
same here. 
Have we not already a func to getProjectVersion that can be used here instead of re-impl a local func?
Also, I think that it should return error and not bool as the command.